indico/htdocs/sass/modules/_bootstrap.scss

```diff
@@ -0,0 +1,262 @@
+@import 'base';
+
+.bootstrap-header {
+    @extend %font-family-bootstrap;
+
+    font-weight: bolder;
+
+    z-index: 1;
+
+    display: block;
+
+    width: 100%;
+    height: 350px;
+    padding-top: 20px;
+
+    text-align: center;
+
```

Comment:
i guess for huge rulesets it could be useful but with so many single-line blocks it looks a bit strainge